stylus/include/OurOfferings.styl

```diff
@@ -0,0 +1,25 @@
+.OurOfferings
+	margin-top: 5rem
+	text-align: center
+	>h1
+		font-family: Arvo
+		margin-bottom: 2rem
+	>p
+		margin-left: 33%
+		margin-right: 33%
+	>div
+		margin-bottom: 5rem
+		margin-left: 20rem
```

Comment:
Avoid fixed sizes like this. They are not responsive, i.e. when you resize the browser, the proportions will not be retained and it will start to look wrong. Percentages are responsive.

Reply:
For margins, I think its okay to use `rem` units.  Percentages get distorted at various sizings.  33% on a large screen vs 33% on a small screen can introduce a non holistic style.

If you're using percentages, you can probably find using CSS flex or grid rules that get you where you want.

Reply:
Media queries often end up being necessary regardless of what you use, but I disagree here, Chris. Percentages retain the proportion of the elements. Rems do not. When you set large margins like Alex is here, you are attempting to place the content at a certain point in relation to the edge of the browser window. If you just use rems, when you resize the browser, the content will no longer be in the same spot. You can see this clearly with the page as it stands. At medium browser width, the food item rows are kind of centered on the page (needs a bit of work), but if you stretch to a larger size, all of a sudden the items are overwhelmingly concentrated on the left because that 20rems is now a smaller proportion of the overall width. Percentages do not have this issue because they are dynamically calculated.

Reply:
Right, I guess I just disagree with percentage margins when worrying about the edge of the browser window -- just use flex or grid.

When using margins to create space around an element, go with `rem`

Reply:
It's kind of tricky. I haven't digested grid yet, but with flex, it might be hard to create what we need here: a thin-ish column of content with lots of white space on the sides. You could probably make the view container have `flex-direction: column` and `align-items: center`, but then for the item rows, you'd probably have to specify a basis so they don't use the whole page's width. I think the simplest solution would just be to set margin percentages on the left and right of the view container to take care of all the white space, then just work with the remaining space that's left.